From 33229d4a64f2ead873c1f6b1bf57e4937fed2e0a Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?=E9=BB=84=E6=B6=9B?= <huangtao@rock-chips.com>
Date: Sat, 30 Jul 2011 22:26:30 +0800
Subject: [PATCH] Revert "USB: gadget: composite: Don't call set_alt() on
 functions that are hidden."

This reverts commit 6fd76fd99efabcc28ca58d32257a608be22e1019.
---
 drivers/usb/gadget/composite.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/drivers/usb/gadget/composite.c b/drivers/usb/gadget/composite.c
index aac724ae664b..c7a802d97e93 100755
--- a/drivers/usb/gadget/composite.c
+++ b/drivers/usb/gadget/composite.c
@@ -477,8 +477,6 @@ static int set_config(struct usb_composite_dev *cdev,
 
 		if (!f)
 			break;
-		if (f->hidden)
-			continue;
 
 		result = f->set_alt(f, tmp, 0);
 		if (result < 0) {
-- 
2.35.3

